src/core/types/exactly_n_bytes.rs

```diff
@@ -103,6 +103,9 @@ macro_rules! decl_exactly_n_bytes {
                 secret_magic: [<Exactly $byte_count Bytes SecretMagic>],
             }
 
+            // Make it JSON String convertible in Swift/Kotlin
+            json_string_convertible!([<Exactly $byte_count Bytes>]);
```

Comment:
A macro calling a macro calling a macro 🤯 

As @micbakos-rdx said, compared to Rust other languages are toy languages 🤩

Reply:
<img src="https://media1.giphy.com/media/bUkXxGkGYb5bq/giphy.gif"/>